packages/flutter_test/lib/src/binding.dart

```diff
@@ -978,6 +976,11 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
     });
   }
 
+  @override
+  void ensureFrameCallbacksRegistered() {
+    // Leave Window alone, do nothing.
```

Comment:
maybe assert that window.onBeginFrame and window.onDrawFrame are null...

Reply:
Done.